recipes/libbezier/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "libbezier" %}
+{% set github_org = "dhermes" %}
+{% set github_repo = "bezier" %}
+{% set version = "2020.2.3" %}
+{% set tar_gz_hash = "d2387f9089d2317a80bc97f278843d5326d1bebde7b08c9c9f6caaed9b0a08b1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ github_org }}/{{ github_repo }}/archive/{{ version }}.tar.gz
+  sha256: {{ tar_gz_hash }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler("fortran") }}
+    - cmake
```

Comment:
Probably need to add make here

Reply:
I'd love some guidance here actually. So I use

```
cmake \
    --build "${BUILD_DIR}" \
    --config Release \
    --target install
```

which means whatever the `CMAKE_GENERATOR` that was used during the configure step will get used during the build / install step.

I've observed

- Windows: `CMAKE_GENERATOR=NMake Makefiles`
- macOS: ??
- Linux: `CMAKE_GENERATOR=Unix Makefiles` (based on [this][1])

However I am manually overriding on Windows to be `-G "MinGW Makefiles"`. So it seems I **do** need to depend on `make` (since `Unix Makefiles` is the default on macOS and Linux) and on Windows I need to depend on **something** that will ensure `mingw` is present. 

After reading a bit more, I think I was doing the "wrong" thing by relying on AppVeyor `%PATH%` manipulation to find `gfortran`. I also now realize what the `*_compiler` suffix actually does with the Jinja2 `compiler()` helper, so that on Windows when it prints

```
...
(base) %SRC_DIR%>set "fortran_compiler=flang" 
...
(base) %SRC_DIR%>set "m2w64_fortran_compiler=m2w64-toolchain" 
(base) %SRC_DIR%>set "CMAKE_GENERATOR=NMake Makefiles" 
...
```

That means I should be using `{{ compiler("m2w64_fortran") }}`.

[1]: https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html